crates/cranelift/src/translate/code_translator.rs

```diff
@@ -2898,54 +2898,150 @@ pub fn translate_operator(
-                "codegen for stack switching instructions not implemented, yet".to_string(),
-            ));
+            let src_types = environ.continuation_arguments(TypeIndex::from_u32(*argument_index));
+            let dst_arity = environ
+                .continuation_arguments(TypeIndex::from_u32(*result_index))
+                .len();
+            let arg_count = src_types.len() - dst_arity;
+
+            let arg_types = &src_types[0..arg_count];
+            for arg_type in arg_types {
+                // We can't bind GC objects using cont.bind at the moment: We
+                // don't have the necessary infrastructure to traverse the
+                // buffers used by cont.bind when looking for GC roots. Thus,
+                // this crude check ensures that these buffers can never contain
+                // GC roots to begin with.
+                if arg_type.is_vmgcref_type_and_not_i31() {
+                    return Err(wasmtime_environ::WasmError::Unsupported(
+                        "cont.bind does not support GC types at the moment".into(),
+                    ));
+                }
+            }
+
+            let (original_contobj, args) = state.peekn(arg_count + 1).split_last().unwrap();
+
+            let new_contobj = environ.translate_cont_bind(builder, *original_contobj, args);
+
+            state.popn(arg_count + 1);
+            state.push1(new_contobj);
         }
-        Operator::Suspend { tag_index: _ } => {
-            // TODO(10248) This is added in a follow-up PR
-            return Err(wasmtime_environ::WasmError::Unsupported(
-                "codegen for stack switching instructions not implemented, yet".to_string(),
-            ));
+        Operator::Suspend { tag_index } => {
+            let tag_index = TagIndex::from_u32(*tag_index);
+            let param_types = environ.tag_params(tag_index).to_vec();
+            let return_types: Vec<_> = environ
+                .tag_returns(tag_index)
+                .iter()
+                .map(|ty| crate::value_type(environ.isa(), *ty))
+                .collect();
+
+            let params = state.peekn(param_types.len());
+            let param_count = params.len();
+
+            let return_values =
+                environ.translate_suspend(builder, tag_index.as_u32(), params, &return_types);
+
+            state.popn(param_count);
+            state.pushn(&return_values);
         }
         Operator::Resume {
-            cont_type_index: _,
-            resume_table: _,
+            cont_type_index,
+            resume_table,
         } => {
-            // TODO(10248) This is added in a follow-up PR
-            return Err(wasmtime_environ::WasmError::Unsupported(
-                "codegen for stack switching instructions not implemented, yet".to_string(),
-            ));
+            // We translate the block indices in the resumetable to actual Blocks.
+            let mut resumetable = vec![];
```

Comment:
Can we name the local something else? Having both a `resume_table` and `resumetable` variable in scope at the same time is pretty confusing. Maybe we can have `wasm_resume_table` and `clif_resume_table`?